js/forum/src/components/EmojiAreaButton.js

```diff
@@ -8,23 +8,18 @@ import icon from "flarum/helpers/icon";
 
 export default class EmojiAreaButton extends Component {
 
-    init() {
-        this.textEditor = null;
```

Comment:
You should keep the init and use `this.textEditor = this.props.textEditor;`

This will insure a text editor is passed to the class.